File: C#/1010.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            double total=0, preco;
            int qtd;
            for (int i=0;i<2;i++)
            {
                string[] vet = Console.ReadLine().Split(' ');
                qtd = Int32.Parse(vet[1]);
                preco = Double.Parse(vet[2]);
                total += qtd * preco;   
            }
            Console.WriteLine("VALOR A PAGAR: R$ "+ string.Format("{0:0.00}", total));
        }
    }
}

File: GO/1010.go
package main

import "fmt"

func main() {
    var a,b int
	var c, r float64
	fmt.Scanln(&a, &b, &c)
    d := float64(b)  
    r = d*c
    fmt.Scanln(&a, &b, &c)
    e := float64(b) 
    r += e*c
	fmt.Printf("VALOR A PAGAR: R$ %.2f\n", r)
}
